site/app/controllers/PollController.php

```diff
@@ -352,6 +367,13 @@ public function submitEdits(): RedirectResponse {
             }
             $file_path = FileUtils::joinPaths($this->core->getConfig()->getCoursePath(), "uploads", "polls", "poll_image_" . $_POST["poll_id"] . "_" . $_FILES["image_file"]["name"]);
             move_uploaded_file($file["tmp_name"], $file_path);
+            $status = FileUtils::validateUploadedFiles($_FILES["image_file"]);
+            if (array_key_exists("failed", $status)) {
```

Comment:
same comment, use `$status['success']` to check if there were any errors and return whatever is set in `$status['error']` if so.